code/lib/types/src/modules/store.ts

```diff
@@ -35,15 +35,18 @@ export type Store_ModuleImportFn = (path: Store_Path) => Store_PromiseLike<Store
 type Store_MaybePromise<T> = Promise<T> | T;
 
 export type Store_TeardownRenderToDOM = () => Store_MaybePromise<void>;
-export type Store_RenderToDOM<TFramework extends AnyFramework> = (
+// We should consider renaming this to `RenderToRootElement` or similar
+export type Store_RenderToDOM<TFramework extends AnyFramework, TRootElement = HTMLElement> = (
```

Comment:
I think it'd be harder to rename this one

Reply:
Does that mean that each renderer also needs to be parameterized somehow?

Reply:
Hmm, I don't think so. I guess a non-web renderer might need to define it's `renderToX` function as a non-web version (i.e. `export const renderToRootElement: RenderToRootElement<XFramework, YRootElement>`) but the default would apply in 99% of cases which do use the web.